### DISCUSSION TITLE
Code Search Engine with ASP.NET Core and Blazor

### DISCUSSION THREAD

**Discussion by bytefish** [microsoft/fluentui-blazor#590] [Show and tell]
I have written a small Blazor Frontend for a Code Search, that I am currently experimenting with:

* https://github.com/bytefish/ElasticsearchCodeSearch/

It uses FluentUI components and I have also used the Pagination component. I had to copy the Pagination Component code and some infrastructure code to get it working, but it all plays nicely together and the UI was only a few lines of code. It's my first Blazor project, so there's a lot of room for improvement I guess.

**Comment by vnbaaij** [comment]
Will do a review after the holidays

**Comment by bytefish** [comment]
@vnbaaij Maybe it's still of interest. I have updated the example to the latest Blazor FluentUI components and it was pretty easy to do. The project is also fully containerized with Docker, so it also shows how to host the Blazor WASM application and serve it using ASP.NET Core.
